src/components/Ffuf/FfufTool.tsx

```diff
@@ -14,7 +14,7 @@ const description_userguide =
     "and directories. This can reveal vulnerabilities in web applications as well generally data-mine " +
     "and map out the target. Ffuf can even be used to brute force credentials used in web authentication.\n\n" +
     "For further information on ffuf: https://github.com/ffuf/ffufS\n\n" +
-    "Wordlist directory: /src-tauri/exploits/ffuf_wordlists/\n\n" +
+    "Wordlist directory: /usr/shar/ddt/ffuf_wordlists/\n\n" +
```

Comment:
Seems to be a spelling error. Should be share instead of "shar"

Reply:
Thanks for catching this! Fixed now.